cmd/create/cluster/cmd.go

```diff
@@ -252,6 +252,24 @@ func run(cmd *cobra.Command, _ []string) {
 		reporter.Errorf("Error getting region: %v", err)
 		os.Exit(1)
 	}
+
+	// Create the AWS client:
+	client, err := aws.NewClient().
+		Logger(logger).
```

Comment:
I have an open pull request https://github.com/openshift/moactl/pull/92 that will add a new `aws.DefaultRegion` (`us-east-1`) which will be used for all requests to the cloudformation API, that way we no longer run into the scenario where a user creates the CF stack in one region, then the client tries to query for the credentials from a non-existent stack in a different region.

Once that's merged, this will need to have an extra `Region(aws.DefaultRegion).` to ensure that it's compatible with the changes in https://github.com/openshift/moactl/pull/92

Reply:
ack @vkareh we can add that once #92 has been merged